sips/sip-137.md

```diff
@@ -0,0 +1,34 @@
+---
+sip: 137
```

Comment:
this number has been taken already - but you can leave this till we finish the review.